src/audio/lib/heap.c

```diff
@@ -1396,6 +1396,9 @@ typedef struct {
     /* 0xC */ u8 newMedium;
 } StorageChange; // size = 0x10
 
+s32 D_801FD120;
+static s32 sBssPad[3];
```

Comment:
I don't see any other bss besides this, so I think this pad is not really required, it should be generated automatically.

Reply:
Not quite. Without the pad bss ends up at `0x10` size, but it needs to be `0x20` size.